recipes/base_install.rb

```diff
@@ -105,13 +105,19 @@
 end
 
 if !node['cfncluster']['custom_node_package'].nil? && !node['cfncluster']['custom_node_package'].empty?
-  execute "install cfncluster-node" do
-    command "sudo pip uninstall --yes cfncluster-node &&" \
-      "cd /tmp &&" \
-      "curl -v -L -o /tmp/cfncluster-node.tgz #{node['cfncluster']['custom_node_package']} &&" \
-      "tar -xzf /tmp/cfncluster-node.tgz &&" \
-      "cd /tmp/cfncluster-node-* &&" \
-      "sudo /usr/bin/python setup.py install"
+  bash "install cfncluster-node" do
+    cwd '/tmp'
+    code <<-EOH
+      if [ "#{node['cfncluster']['cfn_proxy']}" != "NONE" ]; then
+        export http_proxy=#{node['cfncluster']['cfn_proxy']}
```

Comment:
Why is this needed? You can just source `/tmp/proxy.sh`. See https://github.com/awslabs/cfncluster/blob/44ab57d8abd1444d45f5ede0371f6aa96d0c987f/cloudformation/cfncluster.cfn.json#L3187